mono/metadata/w32process.h

```diff
@@ -52,22 +53,55 @@ typedef struct
 	guint32 window_style;
 	MonoBoolean error_dialog;
 	gpointer error_dialog_parent_handle;
-	MonoBoolean use_shell_execute;
-	MonoString *username;
-	MonoString *domain;
-	MonoObject *password; /* SecureString in 2.0 profile, dummy in 1.x */
-	MonoString *password_in_clear_text;
-	MonoBoolean load_user_profile;
-	MonoBoolean redirect_standard_input;
-	MonoBoolean redirect_standard_output;
-	MonoBoolean redirect_standard_error;
-	MonoObject *encoding_stdout;
-	MonoObject *encoding_stderr;
-	MonoBoolean create_no_window;
-	MonoObject *weak_parent_process;
-	MonoObject *envVars;
+	MonoBoolean unused_use_shell_execute;
```

Comment:
Could also remove these lines. Native doesn't need the accurate size, just the offsets/sizes/types of the preceding fields.